src/dmd/toctype.d

```diff
@@ -176,7 +177,7 @@ public:
                 }
             }
 
-            if (global.params.symdebugref)
+            if (global.params.symdebugref && target.os == Target.OS.Windows)
```

Comment:
Why is this individualizing windows?

Reply:
This was a stub function for non windows before. This preserves that behaviour.

Reply:
Ok, but I would like to avoid another conflict about refactorings and code modifications in the same PR. Could you please pull this out and submit it as a separate PR?